packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -63,6 +63,15 @@ export default function SidebarNavigationScreenNavigationMenu() {
 		);
 	}
 
+	if ( ! navigationMenu?.content?.raw ) {
+		return (
+			<SidebarNavigationScreenWrapper
+				title={ decodeEntities( menuTitle ) }
+				description={ __( 'This Navigation Menu is empty.' ) }
```

Comment:
```suggestion
				description={ __( 'This Navigation menu is empty.' ) }
```

Elsewhere we used `Navigation menu`.